assets/javascripts/discourse/templates/components/wizard-custom-action.hbs

```diff
@@ -283,6 +283,62 @@
   </div>
 {{/if}}
 
+{{#if watchCategories}}
+  <div class="setting full field-mapper-setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.action.watch_categories.categories"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{wizard-mapper
+        inputs=action.categories
+        property='categories'
+        onUpdate=(action 'mappedFieldUpdated')
+        options=(hash
+          textSelection='key,value'
+          wizardFieldSelection=true
+          userFieldSelection='key,value'
+          context='action'
+        )}}
+    </div>
+  </div>
+
+  <div class="setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.action.watch_categories.notification_level.label"}}</label>
+    </div>
+
+    <div class="setting-value">
+      {{combo-box
+        value=action.notification_level
+        content=availableNotificationLevels
+        onChange=(action (mut action.notification_level))
+        options=(hash
+          isDisabled=action.custom_title_enabled
+          none='admin.wizard.action.watch_categories.select_a_notification_level'
+        )}}
+    </div>
+  </div>
+
+  <div class="setting">
```

Comment:
We need to either rearrange the order of these inputs or add another class so they align with each other in the admin panel. Currently the inputs look skewed.

<img width="521" alt="Screenshot at May 26 09-43-22" src="https://user-images.githubusercontent.com/5931623/82848542-42259b80-9f37-11ea-8033-ed026a2f9a24.png">

Reply:
Improved.  We might want to review the CSS more deeply.